Mathlib/Data/Bool/AllAny.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2017 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
+
+! This file was ported from Lean 3 source module data.bool.all_any
+! leanprover-community/mathlib commit 5a3e819569b0f12cbec59d740a2613018e7b8eec
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.List.Basic
+
+/-!
+# Boolean quantifiers
+
+This proves a few properties about `list.all` and `list.any`, which are the `bool` universal and
+existential quantifiers. Their definitions are in core Lean.
+-/
+
+
+variable {α : Type _} {p : α → Prop} [DecidablePred p] {l : List α} {a : α}
+
+namespace List
+
+--#print List.all_nil
+/-
+@[simp]
+theorem all_nil (p : α → Bool) : all [] p = tt :=
+  rfl
+#align list.all_nil List.all_nil
+-/
```

Comment:
This is already in `lake-packages/std/Std/Data/List/Init/Lemmas.lean`
Should I just delete those lines then?

Reply:
You should definitely keep the `#align`, even if the lemma is in Std now

Reply:
done, added a porting note saying this is now in Std.